### PR TITLE
[DMC] Implement Tiller Engine

### PR DESCRIPTION
**Pull request by PurpleCrowbar**

### CODE REVIEW

**Comment by PurpleCrowbar** [comment]
[[Tiller Engine]]

File: Mage.Sets/src/mage/cards/t/TillerEngine.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+ */
+public final class TillerEngine extends CardImpl {
+
+    private static final FilterControlledLandPermanent filter = new FilterControlledLandPermanent("a land");
+    private static final FilterPermanent filter2 = new FilterNonlandPermanent("nonland permanent an opponent controls");
+
+    static {
+        filter.add(TappedPredicate.TAPPED);
+        filter2.add(TargetController.OPPONENT.getControllerPredicate());
+    }
```
Comment (awjackson) [inline Mage.Sets/src/mage/cards/t/TillerEngine.java:31]: We have StaticFilters for both of these. `StaticFilters.FILTER_LAND_A` for "a land" and `StaticFilters.FILTER_OPPONENTS_PERMANENT_NON_LAND` for "nonland permanent an opponent controls".

Comment (PurpleCrowbar) [reply Mage.Sets/src/mage/cards/t/TillerEngine.java:28]: Done

**Comment by PurpleCrowbar** [comment]
Should be good to merge, everything looks good on my local test server in terms of text formatting, effect outcomes, etc

**Review by awjackson — APPROVED** [review]

**Pull request merged**